_episodes/03-packaging-installing.md

```diff
@@ -116,129 +117,133 @@ This shows the basic commands and the general options.
+
+
+If a `.py` file is on the path, we can import functions from it to our current file. Open up Python, import `sys` and print the path.
+
+~~~
+import sys
+sys.path
+~~~
+{:.language-python}
 
+sys.path is a list of strings, each describing the absolute path to a directory. Python will look in these directories for modules. If we have a directory containing modules we want Python to be aware of, we append it to the path:
 
-If the `.py` file is on the path, we can import functions from one file to another.
 
-FIXME: example
+~~~
+sys.path.append('/path/to/module/')
+sys.path
+~~~
+{:.language-python}
 
+Once the module you want is in sys.path, it can be imported just like any other module.
 
 ## Python Packages
 
-We don't have to manually add to the path every time though. Instead, we can
-package our modules to be installable, and then we can import directly.  This
-way of importing allows us to better allow python on the backend handle differences in paths across different user systems.  
+To save adding modules to the path every time we want to use them, we can
+package our modules to be installable.  This
+method of importing standardises how we import modules across different user systems.
 
-Packages are namespaces or containers (e.g. a directory) which can contain multiple packages and modules.
+Packages are namespaces or containers which can contain multiple modules. e.g. the `scipy` package contains the `interpolate` and `optimize` modules.
 
-Making python code a package (packaging it) requires no extra tools. We need to:
+Making python code into a package requires no extra tools. We need to
 
-- Create a directory and give it package's name.
-- Put modules in it.
+- Create a directory, named after our package.
+- Put modules (`.py` files) in the directory.
 - Create an `__init__.py` file in the directory
-- Create a setup.py file at the top level
+- Create a `setup.py` file alongside the directory
 
-The `__init__.py` file tells python that the directory is supposed to be read as a package.
+Our final package will look like this:
 
-e.g.
+├── package-name  
+│   ├── \_\_init__.py  
+│   ├── module-a.py  
+│   └── module-b.py  
+└── setup.py
 
-Let us create a package called **Vehicles** with two modules **Land** and **Water**.
+The `__init__.py` file tells python that the directory is supposed to be tread as a package.
+
+Let's create a package called **conversions** with two modules **temp** and **speed**.
 
 ### Step 1: Creating a directory
-Create a directory called **Vehicles**
+Create a directory called **conversions**
 
 ~~~
-mkdir Vehicles
+mkdir conversions
 ~~~
 {: .language-bash}
 
 ### Step 2: Adding Modules
 
+conversions/temp.py
```

Comment:
lets spell out temperature, so that it's not confused with temporary

Reply:
Corrected in e29a1dc